cli/network.go

```diff
@@ -25,6 +24,7 @@ const (
 	UpdateTypeLink  NetlinkUpdateType = "link"
 	UpdateTypeAddr  NetlinkUpdateType = "addr"
 	UpdateTypeRoute NetlinkUpdateType = "route"
+	fakeBridge      string            = "runv0"
```

Comment:
all taps will be added to this bridge. I'm afraid it might cause information leak.

Reply:
when the taps are moved into shim's ns, they are also atomically moved out from the bridge. But what happen before the moving?

Reply:
Have no idea, What's your opinion?